libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/serial_api.c

```diff
@@ -512,7 +512,11 @@ int serial_getc(serial_t *obj) {
     int data;
     int was_masked;
 
+#if defined ( __ICCARM__ )
+    was_masked = __disable_irq_iar();
```

Comment:
I found it. For references, please use file reference here on github, which would show the file caFunc, and this code snippet

```
#elif (defined (__ICCARM__)) /*---------------- ICC Compiler ---------------------*/

#error IAR Compiler support not implemented for Cortex-A
```

I understand now why you implemented your own disable irq. A question is, why cmsis func are not enabled for ICCARM? I am fine with this as the current version does not support it. Do newer versions provide the function?